content/en/docs/concepts/storage/projected-volumes.md

```diff
@@ -71,18 +71,24 @@ A container using a projected volume source as a [`subPath`](/docs/concepts/stor
 volume mount will not receive updates for those volume sources.
 {{< /note >}}
 
-## SecurityContext interactions
+### serviceAccountToken and securityContext
 
 The [proposal](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2451-service-account-token-volumes#proposal) for file permission handling in projected service account volume enhancement introduced the projected files having the the correct owner permissions set.
```

Comment:
We try _not_ to link to enhancements in docs. In my mind it's the difference between giving people a set of changes and giving people the result of applying those changes. Readers are interested mostly in how Kubernetes works now and less in the journey to get here.

Reply:
Agree 100%. Just for some context here, I'm not the author of this document. I'm resolving a bug report for ephemeral containers where everything was working as designed but not as expected by the user.

I learned enough about projected service account tokens to update the part of this document related to the bug, but I don't feel comfortable rewriting other parts of the doc.